ReactWindows/ReactNative.Shared/UIManager/BorderedCanvasManager.cs

```diff
@@ -12,16 +13,58 @@ namespace ReactNative.UIManager
     /// View parent manager for bordered canvases.
     /// </summary>
     public abstract class BorderedCanvasManager<TCanvas> : BorderedViewParentManager<TCanvas>
```

Comment:
>BorderedCanvasManager [](start = 26, length = 21)

Since we're making breaking changes to this, and it's no longer used anywhere, I think it's safe to just delete this abstraction.  IIRC, the abstraction was added to accommodate some third party module.